### PR TITLE
Update baseline Lint rules

### PR DESCRIPTION
**Pull request by MiSikora**
## Description

Updating lint rules due to bugs in lint and checking some inconsistencies between CI and local builds.

Internal ref: p1716983402292699-slack-CC7L49W13

## Testing Instructions

CI checks are enough.

## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [ ] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [ ] I have considered whether it makes sense to add tests for my changes
- [ ] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [ ] Any jetpack compose components I added or changed are covered by compose previews
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
 
#### I have tested any UI changes...

- [ ] with different themes
- [ ] with a landscape orientation
- [ ] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is larger than 500 lines of changes. Please consider splitting it into smaller PRs for easier and faster reviews.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: modules/features/profile/lint-baseline.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
         errorLine2="                 ~~~~~~~~">
         <location
             file="src/main/java/au/com/shiftyjelly/pocketcasts/profile/cloud/CloudFilesFragment.kt"
-            line="158"
+            line="156"
```
Comment (MiSikora) [inline modules/features/profile/lint-baseline.xml:18]: I have no clue where changes in this file came from.

File: modules/services/images/lint-baseline.xml [hunk 2]
```diff
@@ -752,14 +752,6 @@
...
         <location
             file="src/main/res/drawable-xxxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-xxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-xxhdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-mdpi/appicon_patron_dark.png"/>
-        <location
-            file="src/main/res/drawable-hdpi/appicon_patron_dark.png"/>
```
Comment (wzieba) [inline modules/services/images/lint-baseline.xml:762]: This is removed by the lint update baseline, or manually?

Comment (MiSikora) [reply modules/services/images/lint-baseline.xml:762]: When generating. And it doesn't fail now unlike previous times: https://buildkite.com/automattic/pocket-casts-android/builds/6741#018fc355-4cd2-4c4b-909a-586ad829a3e0

Comment (MiSikora) [reply modules/services/images/lint-baseline.xml:762]: Oh wow… it is sooo weird. @ashiagr generated baseline rules in her PR as well [here](https://github.com/Automattic/pocket-casts-android/pull/2279/commits/b2f7baf79862c14158fb5d765e503078a35ec111). And it is different from what is generated on my machine. I've checked out her branch and generated again and this is the diff.

[baseline.patch](https://github.com/Automattic/pocket-casts-android/files/15485200/baseline.patch)

**Review by wzieba — APPROVED** [review]

**Pull request merged**